docs/articles/documentation/examples/README.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Examples
+permalink: /documentation/examples/
+---
+# Examples
+
+This section lists examples from the [testcafe-examples](https://github.com/DevExpress/testcafe-examples) repository. All examples are ready to run.
+
+## Locating Elements
+
+Example | Description
+------- | ------------
+[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Find a header element with extra space characters by trimmed text content.
+[Use XPath Selectors](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-xpath-selectors) | Find an input with an XPath selector.
+
+## Testing the DOM
+
+Example | Description
+------- | ------------
+[Access Element Properties](https://github.com/DevExpress/testcafe-examples/tree/master/examples/element-properties) | Access DOM element's properties and verify them with assertions.
+[Iterate Over Table Rows](https://github.com/DevExpress/testcafe-examples/tree/master/examples/iterate-over-table-rows) | Verify cell content for even and odd rows.
+[Test \<select\> Elements](https://github.com/DevExpress/testcafe-examples/tree/master/examples/test-select-elements) | Open a `<select>` element's drop-down list, click an item, and verify the value.
+[Page Reload](https://github.com/DevExpress/testcafe-examples/tree/master/examples/page-manipulation) | Execute custom code on the page to obtain data or manipulate the page state.
+
+## Test Organization
+
+Example | Description
+------- | ------------
+[Extract Code to Helpers](https://github.com/DevExpress/testcafe-examples/tree/master/examples/extract-code-to-helpers) | Extract test code to functions defined in a separate module.
+[Create Data-Driven Tests](https://github.com/DevExpress/testcafe-examples/tree/master/examples/create-data-driven-tests) | Create a test that performs parameterized actions.
+[Use Page Model](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-page-model) | Abstract out element selectors and common operations with these elements into a page model.
+
+## Interoperability
+
+Example | Description
+------- | ------------
+[Inject Custom Client Scripts](https://github.com/DevExpress/testcafe-examples/tree/master/examples/client-scripts) | Inject Node.js modules into the tested webpage.
+[Import Third-Party Modules](https://github.com/DevExpress/testcafe-examples/tree/master/examples/import-third-party-modules) | Import a third-party Node.js module into test code.
+
+## Browser API Mocks
+
+Example | Description
+------- | ------------
+[Mock Date](https://github.com/DevExpress/testcafe-examples/tree/master/examples/mock-date) | Use [mockdate](https://www.npmjs.com/package/mockdate) to substitute the current data.
+[Mock Geolocation API](https://github.com/DevExpress/testcafe-examples/tree/master/examples/mock-geolocation-api) | Substitute the location in test code.
```

Comment:
If you say "Substitute" in this sentence, then we also need to clarify what is being substituted.
Do you mean:

"Substitutes the current location with a mock location."